ITE/2/README.adoc

```diff
@@ -169,21 +169,23 @@ are three important considerations regarding the targets metadata signed
 by the delegated "`package-and-in-toto-metadata-signer`" role:
 
 [arabic]
-. Since each package is typically produced independently of other
-packages, this means that the complete set of in-toto _link_ metadata
-for a package SHOULD be different, and therefore isolated, from that for
-another package. (The in-toto _root layout_ metadata SHOULD be the same
-for every package.) This MAY be done in a number of different ways, but
-perhaps the simplest is to use directories with unique names to isolate
-different in-toto link metadata for different packages, where the name
-MAY be the SHA-256 hash of the "`developer`" step link metadata file for
-a particular package.
+. Since each package for a project is typically produced independently
+of other packages, this means that the complete set of in-toto _link_
+metadata for a package SHOULD be different, and therefore isolated,
+from that for another package. This MAY be done in a number of different
+ways, but perhaps the simplest is to use directories with unique names to
+isolate different in-toto link metadata for different packages, where the
+name MAY be the SHA-256 hash of the "`developer`" step link metadata file
+for a particular package. The root layout MAY be identical for each
+of these packages, but it can differ to allow for changes in the supply
```

Comment:
Could we please mention here that using consistent snapshots and custom targets metadata in TUF lets us trivially associate different packages (whether in the same project or not) with different root layout w/o breaking backwards compatibility?

Reply:
Does https://github.com/trishankatdatadog/ITE/pull/3/commits/df8c94889d1605bd49697d266db18a83a161abee look ok?

Reply:
Actually not so much I think. Let me try again.